interface/patient_file/history/encounters.php

```diff
@@ -41,6 +42,13 @@
     require_once("$srcdir/group.inc");
 }
 
+
+if($GLOBALS['enable_group_therapy'] && !acl_check("groups","glog",false, array('view','write'))){
```

Comment:
Disabling access completely if group therapy is enabled and no glog permission? I don't understand this.